README-trust.md

```diff
@@ -0,0 +1,116 @@
+Trust module
+============
+
+Description
+-----------
+
+The trust module allows to ensure presence and absence of a domain trust.
+
+Features
+--------
+* Trust management
+
+
+Supported FreeIPA Versions
+--------------------------
+
+FreeIPA versions 4.4.0 and up are supported by the ipagroup module.
+
+
+Requirements
+------------
+
+**Controller**
+* Ansible version: 2.8+
+
+**Node**
+* Supported FreeIPA version (see above)
+
+
+Usage
+=====
+
+Example inventory file
+
+```ini
+[ipaserver]
+ipaserver.test.local
+```
+
+
+Example playbook to create a one-way trust:
+
+```yaml
+---
+- name: Playbook to create a trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: create a one-way trust to the windows.local domain
+    ipatrust:
+      realm: windows.local
+      realm_admin: Administrator
+      realm_passwd: secret_password
+      state: present
+```
+
+Example playbook to create a two-way trust using a shared-secret:
+
+```yaml
+---
+- name: Playbook to create a trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: create a two-way trust to the windows.local domain
+    ipatrust:
+      realm: windows.local
+      trust_secret: my_share_Secret
+      biderectional: True
+      state: present
+```
+Example playbook to delete a trust:
+
+```yaml
+---
+- name: Playbook to delete trust
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: delete the trust with windows.local
+    ipatrust:
+      realm: windows.local
+      state: absent
+```
+This will only delete the ipa-side of the trust and it does NOT delete the id-range that matches the trust,  
+
+Variables
+=========
+
+ipatrust
+-------
+
+Variable | Description | Required
+-------- | ----------- | --------
+`ipaadmin_principal` | The admin principal is a string and defaults to `admin` | no
+`ipaadmin_password` | The admin password is a string and is required if there is no admin ticket available on the node | no
+`realm` | The realm name string. | yes
+`realm_admin` | Active Directory domain administrator string. | no
+`realm_passwd` | Active Directory domain administrator's password string. | no
+`realm_Server` | Domain controller for the Active Directory domain string. | no
```

Comment:
The variable names, in this case, should match CLI parameters.

Reply:
I've implemented your suggestions